captum/attr/_core/feature_ablation.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python3
+
+import torch
+
+from .._utils.common import (
+    _format_attributions,
+    format_input,
+    _run_forward,
+    _expand_additional_forward_args,
+    _format_additional_forward_args,
+)
+from .._utils.attribution import PerturbationAttribution
+
+
+class FeatureAblation(PerturbationAttribution):
```

Comment:
Yes, good point! Since I'm also adding other perturbation based algos, I was going to add a section for these in the readme and website, will add them together in a separate PR if that's ok?